olive/engine/engine.py

```diff
@@ -621,6 +627,22 @@ def get_pareto_frontier_footprints(
 
         return pf_footprints
 
+    def dump_run_history(self, run_history, output_path: str = None):
+        if not run_history:
+            logger.info("No run history to dump!")
+            return
+        headers = run_history[0]._fields
+        try:
+            from tabulate import tabulate
+
+            formatted_rls = tabulate([tuple(rh) for rh in run_history], headers=headers, tablefmt="grid")
+            logger.info(f"run history:\n{formatted_rls}")
+        except ImportError:
+            logger.error("Please install tabulate for better run history output")
```

Comment:
warning?

Reply:
Since we are calling ` dump_run_history` for all runs and tabulate is optional, I think `info` might be better? Otherwise, we are giving error/warning over an optional dependency.